### PR TITLE
Documenting QA process and guidelines

### LINKED ISSUE #14: Start Quality Assurance guide
The idea here is to have the first version of the quality assurance guide. In my Office Hour me and @rpattath came up with a plan for it so this should be put into public more accessible document than the "Community Open Session Minutes" document.

### PR DESCRIPTION
**Pull request by rpattath**
Documenting QA process and guidelines with information for all repos under AnitaB.org.

This PR fixes #14

### CODE REVIEW

File: quality-assurance.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<h1>Pre-requisites for contributing towards Quality Assurance</h1>
+
+1. Join #quality-assurance stream on Zulip. Tag @admins on Zulip to add you under the @qa-team tag.
```
Comment (isabelcosta) [inline quality-assurance.md:3]:
```suggestion
1. Join [#quality-assurance](https://anitab-org.zulipchat.com/#narrow/stream/216325-quality-assurance) stream on Zulip. Tag @admins on Zulip to add you under the @qa-team tag.
```

File: quality-assurance.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      * Mentorship Flutter:
+        * [Dev Environemnt Setup demo](https://anitab.zoom.us/rec/share/6MNVAO7pzE9OfNLP1m6EcJQrT6TgX6a81iRIrqAExUzyft37kQpq80KDeueJJKjY?startTime=1589108623000)
+        * [Dev Environment Setup Notes](https://docs.google.com/document/d/1cLznYyN5VfVraN3swl81Yum1IXG3cyLyIdD85QOuGrY/edit?usp=sharing)
+        * [PR testing demo](https://anitab.zoom.us/rec/play/vsEpIe2rpj03T9aSuQSDA_94W9W0J62shnAXqPINnUnmUXEHMAf0MOEVMOt-f8i1epVN8HeS5BurDHjH?startTime=1593537676000&_x_zm_rtaid=bRSfMzm_T4G4tNo7nBPtXQ.1593605707752.e181143531eb9176c6de82c23bf4f24a&_x_zm_rhtaid=490)
+      * Mentorship Backend:
+        * Dev Env Setup demo - [Recoreding 1](https://anitab.zoom.us/rec/share/2uNHIJT-5EVJS7PzxVrWfK0oQtS1eaa81HdP-vVcyoErlalT9Mv00cK-ZvtOiMk?startTime=1589121214000), [Recording 2](https://anitab.zoom.us/rec/share/2uNHIJT-5EVJS7PzxVrWfK0oQtS1eaa81HdP-vVcyoErlalT9Mv00cK-ZvtOiMk?startTime=1589121947000), [Recording 3](https://anitab.zoom.us/rec/share/2uNHIJT-5EVJS7PzxVrWfK0oQtS1eaa81HdP-vVcyoErlalT9Mv00cK-ZvtOiMk?startTime=1589122325000)
```
Comment (isabelcosta) [inline quality-assurance.md:27]:
```suggestion
        * Dev Env Setup demo - [Recording 1](https://anitab.zoom.us/rec/share/2uNHIJT-5EVJS7PzxVrWfK0oQtS1eaa81HdP-vVcyoErlalT9Mv00cK-ZvtOiMk?startTime=1589121214000), [Recording 2](https://anitab.zoom.us/rec/share/2uNHIJT-5EVJS7PzxVrWfK0oQtS1eaa81HdP-vVcyoErlalT9Mv00cK-ZvtOiMk?startTime=1589121947000), [Recording 3](https://anitab.zoom.us/rec/share/2uNHIJT-5EVJS7PzxVrWfK0oQtS1eaa81HdP-vVcyoErlalT9Mv00cK-ZvtOiMk?startTime=1589122325000)
```

**Comment by rpattath** [comment]
@isabelcosta I have added your comment and also included information from [here](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/test-pr-guide.md) to this doc. Please take a second look. Thanks in advance :)

File: quality-assurance.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+<h1>Pre-requisites for contributing towards Quality Assurance</h1>
```
Comment (isabelcosta) [inline quality-assurance.md:1]:
```suggestion
<h1>Quality Assurance<h1>

<h2>Pre-requisites for contributing towards Quality Assurance</h2>
```

File: quality-assurance.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+       _Screenshot/gif_:  
+       _Expected Result_:  
+       _Actual Result_:
+    
+  4. Additional Comments:
+
+  5. Status of PR Changed to: Needs Review or Ready to Merge.
+  ```
+
+<h1>Quality Assurance Guidelines</h1>
```
Comment (isabelcosta) [inline quality-assurance.md:84]:
```suggestion
<h2>Quality Assurance Contribution Guidelines</h2>
```

**Review by isabelcosta — CHANGES_REQUESTED** [review]
This looks awesome @rpattath , I just made some comments about the hierarchy presentation for this to have 1 title, and then the subsections. Can you please add an entry to this document on the README, next to "our blog" entry. Other than that I am happy to approve

**Review by siddhipandare — APPROVED** [review]

**Comment by siddhipandare** [comment]
Great Job @rpattath 😄 ! Thought QA was difficult to get started with, this document helps a lot to debunk that!

**Comment by isabelcosta** [comment]
@siddhipandare thank you for reviewing this 🤗

**Pull request merged**